templates/base.html

```diff
@@ -25,6 +25,7 @@
     {% if config_identity.custom_css %}
       <style>{{ config_identity.custom_css }}</style>
     {% endif %}
+    {% include "_opengraph.html" %}
```

Comment:
Use a block here with the generic opengraph contained within. E.g.
```
{% block opengraph %}
    {% include "_opengraph.html" og_data=opengraph %}
{% endblock %}
```
Pages that need to disable opengraph would include an empty block `{% block opengraph %}{% endblock %}`.

Actor pages would mimic the base's block above with `og_data=post.to_opengraph_dict`.

Reply:
Nod, I more or less did that, but inside of `_opengraph.html` (so I wouldn't clutter base as much). Do you prefer the block to be in base rather than the include?

Reply:
Change this to the example I put above:
```
{% block opengraph %}
    {% include "_opengraph.html" og_data=opengraph %}
{% endblock %}
```

In `templates/identity/view.html`:
```
{% block opengraph %}
    {% include "_opengraph.html" og_data=identity.to_opengraph_dict %}
{% endblock %}
```

In `templates/activities/post.html`:
```
{% block opengraph %}
    {% include "_opengraph.html" og_data=post.to_opengraph_dict %}
{% endblock %}
```